lib/kube/proxy/resource_rbac_test.go

```diff
@@ -285,6 +311,34 @@ func TestListPodRBAC(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "list default namespace pods for user with traits for default namespace",
+			args: args{
+				user:      userWithTraits,
+				namespace: metav1.NamespaceDefault,
+			},
+			want: want{
+				listPodsResult: []string{
+					"default/nginx-1",
+					"default/nginx-2",
+					"default/test",
+				},
+			},
+		},
+		{
+			name: "list pods in every namespace for user with default namespace traitsck",
+			args: args{
+				user:      userWithTraits,
+				namespace: metav1.NamespaceAll,
+			},
+			want: want{
+				listPodsResult: []string{
+					"default/nginx-1",
+					"default/nginx-2",
+					"default/test",
+				},
+			},
```

Comment:
this test has typo and wants the same results as the previous testcase? not sure if copy/paste error

Reply:
The test is correct. The traits expansion grants access to the default namespace only and the user can only see all pods in `default` namespace. The previous test had a hardcoded namespace def.